.github/workflows/on_push-release.yml

```diff
@@ -97,8 +97,10 @@ jobs:
         id: bundle-release
         uses: ./config/actions/gradle-assemble-and-bundle
         with:
+          access-token: ${{ secrets.PACKAGES_PAT }}
           flavors: 'staging,production'
 #          flavors: 'staging,production'
```

Comment:
could boy scout and remove this comment while we're here?

Reply:
Not entirely sure why that comment is there.... I'll remove it

Reply:
Actually... that comment _should_ remain; the line above needs to change to:

```
flavors: 'staging'
```

Reply:
...because we've not _yet_ got the production app setup in the play store console (it really helps if I remember to add context!)